src/common/search-store.ts

```diff
@@ -3,9 +3,34 @@ import { dockStore } from "../renderer/components/dock/dock.store";
 import { autobind } from "../renderer/utils";
 
 export class SearchStore {
-  @observable searchQuery = ""; // Text in the search input
-  @observable occurrences: number[] = []; // Array with line numbers, eg [0, 0, 10, 21, 21, 40...]
-  @observable activeOverlayIndex = -1; // Index withing the occurences array. Showing where is activeOverlay currently located
+  /**
+   * An utility methods escaping user string to safely pass it into new Regex(variable)
+   * @param value Unescaped string
+   */
+  public static escapeRegex(value?: string): string {
+    return value ? value.replace(/[\-\[\]{}()*+?.,\\\^$|#\s]/g, "\\$&") : "";
+  }
+
+  /**
+   * Text in the search input
+   *
```

Comment:
We can remove such empty lines, since we're not using this style in the codebase.

Reply:
Do you mean between the line that begins "Text in ..." and the "@observable"? See `src/main/cluster.ts:239`